airflow/providers/google/cloud/hooks/automl.py

```diff
@@ -499,9 +481,6 @@ def update_dataset(
         :param update_mask: The update mask applies to the resource.  If a dict is provided, it must
             be of the same form as the protobuf message FieldMask.
         :type update_mask: Union[dict, FieldMask]
-        :param project_id: ID of the Google Cloud project where dataset is located if None then
```

Comment:
While this is correct (seems that update_dataset does not need project_id) I think it should have an entry in UPDATING.md ?

Reply:
@potiuk no need I think, automl is master only

Reply:
Right !